README.md

```diff
@@ -154,7 +154,8 @@ Table of Contents
   * [bitbucket.org](https://bitbucket.org/) — Unlimited public and private Git repos for up to 5 users with Pipelines for CI/CD
   * [chiselapp.com](http://chiselapp.com/) — Unlimited public and private Fossil repositories
   * [codebasehq.com](https://www.codebasehq.com/) — One free project with 100 MB space and 2 users
-  * [codeberg.org](https://codeberg.org/) - Unlimited public and private Git repos
+  * [codeberg.org](https://codeberg.org/) - Unlimited public and private Git repos for free and open-source projects.
+    - Static website hosting with [Codeberg Pages](https://codeberg.page/).
```

Comment:
please keep it one line, thanks

Reply:
Is the static site hosting supposed to be on the same line? I followed the same formatting as the GitHub/GitLab entries.

Reply:
Those have many items and so justify multiple lines.  Multi line entries are a pain I prefer not to have any so it's easy to sort the list, we only add them when a single line is just too unwieldy which this would not be.